src/main/java/teammates/ui/pagedata/InstructorSearchPageData.java

```diff
@@ -55,49 +55,49 @@ public void init(FeedbackResponseCommentSearchResultBundle frcSearchResultBundle
 
         this.searchKey = searchKey;
 
-        this.isSearchCommentForResponses = isSearchCommentForResponses;
+        this.isSearchFeedbackSessionData = isSearchCommentForResponses;
```

Comment:
Change the name of argument passed to `init()` function

`this.isSearchFeedbackSessionData = isSearchCommentForResponses;` -> `this.isSearchFeedbackSessionData = isSearchFeedbackSessionData;`

Reply:
Done